Pointers/UniquePtr.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+#include <iostream>
+
+using std::cout;
+using std::endl;
+
+template<typename T>
+class UniquePtr {
+public:
+	UniquePtr(T* ptr);
+	~UniquePtr();
+	// Don't allow copy
+	UniquePtr(const UniquePtr<T>& other) = delete;
+	UniquePtr<T>& operator=(const UniquePtr<T>& other) = delete;
+	// Allow move
+	UniquePtr(UniquePtr<T>&& other) noexcept;
```

Comment:
`noexcept` specifier is not recommended because if the function does end up throwing an exception the program will call `abort`